proposals/distribution.md

```diff
@@ -0,0 +1,66 @@
+# Abstract
+
+The Docker registry protocol has become the defacto standard across the container registry world ([https://github.com/docker/distribution/blob/master/docs/spec/api.md](https://github.com/docker/distribution/blob/master/docs/spec/api.md)).
+
+In the OCI, having a solid, common distribution specification with conformance testing will ensure long lasting security and interoperability throughout the container ecosystem.
+
```

Comment:
+ This proposal also provides the container ecosystem with a means to discuss and schedule extensions to the distribution specification.